modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-reducing-resource-usage-with-olm-pprof_{context}"]
+= Operator Lifecycle Manager
+
+{sno-caps} clusters that run DU workloads require consistent access to CPU resources. Operator Lifecycle Manager (OLM) periodically gathers performance data from Operators, which causes a spike in CPU usage. The following `ConfigMap` custom resource (CR) disables the gathering of Operator performance data by OLM.
+
+.Recommended cluster OLM configuration (`ReduceOLMFootprint.yaml`)
```

Comment:
(`ReduceOLMFootprint.yaml`) is this required?

Reply:
Yes - it links to a file in a recommend cluster spec container